osbenchmark/workload_generator/workload_generator.py

```diff
@@ -123,16 +136,36 @@ def create_workload(cfg):
 
     logger.info("Creating workload [%s] matching indices [%s]", workload_name, indices)
     logger.info("Number of Docs: %s", number_of_docs)
-    client = OsClientFactory(hosts=target_hosts.all_hosts[opts.TargetHosts.DEFAULT],
-                             client_options=client_options.all_client_options[opts.TargetHosts.DEFAULT]).create()
+    client = OsClientFactory(
+        hosts=target_hosts.all_hosts[opts.TargetHosts.DEFAULT],
+        client_options=client_options.all_client_options[opts.TargetHosts.DEFAULT],
+    ).create()
 
     info = client.info()
-    console.info(f"Connected to OpenSearch cluster [{info['name']}] version [{info['version']['number']}].\n", logger=logger)
+    console.info(
+        f"Connected to OpenSearch cluster [{info['name']}] version [{info['version']['number']}].\n",
+        logger=logger,
+    )
+
+    output_path = os.path.abspath(
+        os.path.join(io.normalize_path(root_path), workload_name)
+    )
+
+    operations_path = os.path.join(output_path, "operations")
+    test_procedures_path = os.path.join(output_path, "test_procedures")
+
+    try:
+        shutil.rmtree(output_path)
```

Comment:
To help readers understand this section quickly, would recommend inserting a logging statement here such as:
```
logger.info(f"Removing existing workload [{workload_name}] in path [{output_path}]")
```